targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L452xE/TOOLCHAIN_ARM/stm32l452xe.sct

```diff
@@ -68,7 +68,7 @@ LR_IROM1  MBED_APP_START  MBED_APP_SIZE  {
    .ANY (+RW +ZI)
   }
 
-  ARM_LIB_HEAP  AlignExpr(+0, 16)  EMPTY  (MBED_RAM_SIZE-RAM_FIXED_SIZE+MBED_RAM_START-AlignExpr(ImageLimit(RW_IRAM1), 16))  { ; Heap growing up
+  ARM_LIB_HEAP  AlignExpr(+0, 16)  EMPTY  (MBED_RAM_SIZE-RAM_FIXED_SIZE+MBED_IRAM2_START-AlignExpr(ImageLimit(RW_IRAM1), 16))  { ; Heap growing up
```

Comment:
Similarly, this targets has two RAMs at two distinct locations: https://github.com/ARMmbed/mbed-os/blob/mbed-os-6.6.0/targets/TARGET_STM/TARGET_STM32L4/TARGET_STM32L452xE/cmsis_nvic.h

See my other comment. I think the old calculation is correct here.
```suggestion
  ARM_LIB_HEAP  AlignExpr(+0, 16)  EMPTY  (MBED_RAM_SIZE-RAM_FIXED_SIZE+MBED_RAM_START-AlignExpr(ImageLimit(RW_IRAM1), 16))  { ; Heap growing up
```